cap-primitives/src/yanix/fs/mod.rs

```diff
@@ -1,6 +1,7 @@
 mod dir_entry_inner;
 mod dir_options_ext;
 mod dir_utils;
+pub(crate) mod errors;
```

Comment:
Oh, that’s a good catch, and yes, i believe what you said is accurate.

Reply:
I've redone it slightly so that we don't get awkward "export one module under two different names situation" but this requires some `cfg_if` magic in `crate::fs::errors` module.